### ISSUE TITLE
`php manage.php` broken

### ISSUE THREAD

**Issue by Giszmo** [osTicket/osTicket#5629]
### Prerequisites

* [ ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [X] Do you have any errors in the PHP error log, or javascript console?
* [X] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [X] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/osTicket/osTicket/wiki/Github-Issues-Guidelines)

### Description

Running `# php manage.php` yields error where before I got some "help".

### Steps to Reproduce

1. check out revision 41e0c6c0 (current HEAD on branch `develop`)
2. run `php manage.php`

**Expected behavior:**

Whatever it did in earlier versions ...

```
# git checkout v1.12.6
# php manage.php
Error: action is a required argument

Manage one or more osTicket installations

Usage:
    manage.php manage.php action [options] [arguments]

Options:
    -h, --help          Display this help message

Arguments:
action
    Action to be managed


Currently available modules follow. Use 'manage.php <module> --help' for
usage regarding each respective module:

agent    CLI agent manager
cron     CLI cron manager for osTicket
deploy   Deploys osTicket into target install path
export   Dumps the osTicket database in formats suitable for the importer
file     CLI file manager for osTicket
i18n     Manages translation files from Crowdin
import   Imports data from a previous backup (using the exporter)
list     CLI list manager
org      CLI organization manager
package  Creates an osTicket distribution ZIP file
serve    Run a CLI server for osTicket
unpack   Unpacks osTicket into target install path
upgrade  Upgrade an osTicket help desk
user     CLI user manager
```

**Actual behavior:**

```
# git checkout v1.14.3
# php manage.php
Error: action is a required argument

PHP Fatal error:  Cannot declare class Exporter, because the name is already in use in /root/osTicket/include/class.export.php on line 374

Fatal error: Cannot declare class Exporter, because the name is already in use in /root/osTicket/include/class.export.php on line 374
```
### Versions

I try to upgrade from `v1.12.5-121-g609b8d88` on my 

```
# uname -a
Linux support1 5.4.0-45-generic #49-Ubuntu SMP Wed Aug 26 13:38:52 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by JediKev** [comment]
@Giszmo 

Thank you for the report. Below is the patch to address the issue:
- https://github.com/osTicket/osTicket/pull/5631

Cheers.

**Comment by Giszmo** [comment]
You'll have to close the issue. I dipped my toes and killed the server meanwhile.

**Comment by JediKev** [comment]
Addressed with #5631
